packages/phpbb3.php

```diff
@@ -501,10 +503,14 @@ protected function ExportUserNotes() {
             'log_data' => array(
                 'Column' => 'Body',
                 'Type' => 'text',
-                'Filter' => function ($value) {
-                    $value = @unserialize($value);
+                'Filter' => function ($value, $field, $row) use (&$corruptedRecords) {
+                    $unserializedValue = @unserialize($value);
 
-                    return array_pop($value);
+                    if (!$unserializedValue) {
```

Comment:
Since `array_pop` is used based on whether or not this condition passes, it should probably be checking with `is_array`, instead of just a falsy value.  If `unserialize` fails, it'll return a false, but we don't know for sure what's been successfully unserialized.  It's almost certainly an array, *but* given some of the things I've seen in databases, that isn't a given.